ironic/drivers/modules/drac/raid.py

```diff
@@ -645,18 +682,18 @@ def _assign_disks_to_volume(logical_disks, physical_disks_by_type,
          interface_type, size_mb), disks in physical_disks_by_type.items():
 
         if ('disk_type' in logical_disk
-            and logical_disk['disk_type'] != disk_type):
+                and logical_disk['disk_type'] != disk_type):
```

Comment:
Upstream reviewers will probably ask you to revert this change since it's outside the scope of this patch.

Reply:
Done